### ISSUE TITLE
Will this delete my messages over the 2 week bot limit?

### ISSUE THREAD

**Issue by ericblank** [Fabricio20/DiscordSelfBot#20]
I'm told selfbots can do that

**Comment by Fabricio20** [comment]
Anything can delete messages older than two weeks as soon as it's not using the bot-only Bulk Delete.

(Correct me if I'm wrong @DV8FromTheWorld)

**Comment by DV8FromTheWorld** [comment]
Correct. You can always delete messages so long as you still have access to the channel. The only restriction is when using the BOT only BulkDelete endpoint which is limited to messages younger then 2 weeks
